extensions/shared/components/audio-player/style.scss

```diff
@@ -0,0 +1,215 @@
+@import '../../styles/gutenberg-base-styles.scss';
+
+//Set the default values
+$jetpack-audio-player-primary: $black;
+$jetpack-audio-player-secondary: $gray-400;
+$jetpack-audio-player-background: $white;
+
+//Bit of a hack to check for the existence of the podcast player values
+$jetpack-podcast-player-primary: false !default;
```

Comment:
I'm not sure if this is necessary. Can we always rely on the existence of the podcast player variables? Our defaults could get out of sync with the podcast player defaults. The alternative would be to move these to a shared file and have the podcast player use the audio player values.